src/main/java/org/assertj/core/api/AbstractMapAssert.java

```diff
@@ -962,6 +962,37 @@ public SELF containsOnlyKeys(@SuppressWarnings("unchecked") K... keys) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual map contains only the given keys and nothing else, in any order.
+   *
+   * <p>
+   * Examples :
+   * <pre><code class='java'> Map&lt;Ring, TolkienCharacter&gt; ringBearers = new HashMap&lt;&gt;();
+   * ringBearers.put(nenya, galadriel);
+   * ringBearers.put(narya, gandalf);
+   * ringBearers.put(vilya, elrond);
+   * ringBearers.put(oneRing, frodo);
+   *
+   * // assertion will pass
+   * List&lt;Ring&gt; keysToCheck = Arrays.asList(oneRing, nenya, narya, vilya);
+   * assertThat(ringBearers).containsOnlyKeys(keysToCheck);
+   *
+   * // assertion will fail
+   * keysToCheck = Arrays.asList(oneRing, nenya);
+   * assertThat(ringBearers).containsOnlyKeys(keysToCheck);</code></pre>
```

Comment:
add an example where `keysToCheck` has keys not not in `ringBearers`

Reply:
done